### PR TITLE
doing homework 14

### PR DESCRIPTION
**Pull request by Hromy-Ann**

### CODE REVIEW

File: src/test/java/com/demoqa/MyFirstUITest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class MyFirstUITest {
+    WebDriver driver;
+    JavascriptExecutor js;
+
+    @Test
+    void clickFormsCard() {
+        WebElement formsCard = driver.findElement(By.xpath("//div[@class='card-body']/h5[text()='Forms']"));
```
Comment (alexshamrai) [inline src/test/java/com/demoqa/MyFirstUITest.java]: Move all locators `By.xpath("//div[@class='card-body']/h5[text()='Forms']")` to constants

File: src/test/java/com/demoqa/MyFirstUITest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        WebElement elementsCard = driver.findElement(By.xpath("//div[@class='card-body']/h5[text()='Elements']"));
+        js.executeScript("arguments[0].scrollIntoView(true)", elementsCard);
+        elementsCard.click();
+        WebElement textBoxMenuItem = driver.findElement(By.xpath("//span[text()='Text Box']"));
+        textBoxMenuItem.click();
+        WebElement userEmailInput = driver.findElement(By.id("userEmail"));
+        String inputText = "a@g.com";
+        userEmailInput.sendKeys(inputText);
+        WebElement submitButton = driver.findElement(By.id("submit"));
+        js.executeScript("arguments[0].scrollIntoView(true)", submitButton);
```
Comment (alexshamrai) [inline src/test/java/com/demoqa/MyFirstUITest.java]: Do you really need to do all the clicks via Javascript?

**Review by alexshamrai — CHANGES_REQUESTED** [review]

File: src/test/java/com/demoqa/MyFirstUITest.java (hunk 1 above)
Comment (Hromy-Ann) [reply src/test/java/com/demoqa/MyFirstUITest.java]: done

File: src/test/java/com/demoqa/MyFirstUITest.java (hunk 2 above)
Comment (Hromy-Ann) [reply src/test/java/com/demoqa/MyFirstUITest.java]: Yes, ad has appeared and I need to scroll for my test passing.

**Review by alexshamrai — APPROVED** [review]

**Pull request merged**